src/test/javascript/spec/component/modeling-assessment/modeling-assessment.component.spec.ts

```diff
@@ -215,6 +221,43 @@ describe('ModelingAssessmentComponent', () => {
         expect(relationship!.highlight).to.not.exist;
     });
 
+    it('should update highlighted assessments first round', () => {
+        const changes = { highlightDifferences: { currentValue: true } as SimpleChange };
+        comp.highlightDifferences = true;
+        comp.model = mockModel;
+        fixture.detectChanges();
+        comp.feedbacks = [mockFeedbackWithReference];
+        comp.referencedFeedbacks = [mockFeedbackWithReference];
+
+        comp.ngOnChanges(changes);
+
+        expect(comp.apollonEditor).to.exist;
+        const apollonModel = comp.apollonEditor!.model;
+        const assessments: any = apollonModel.assessments;
+        expect(assessments[0].labelColor).to.equal('#ffa561');
```

Comment:
See my comment about color and magic numbers above.

Reply:
Done